### PR TITLE
Added alter_type function for PG migration

### PR DESCRIPTION
**Pull request by sofakingworld**
Hi! I'm little bit tired of defining migrations every time on enum change, so here i added alter_type function.

Hope it will be helpful for others :)

### CODE REVIEW

**Comment by sofakingworld** [comment]
@gjaldon react please :3

**Comment by sofakingworld** [comment]
@gjaldon may you accept this PR?

File: lib/ecto_enum/postgres/use.ex [hunk 1]
```diff
@@ -77,10 +79,18 @@ defmodule EctoEnum.Postgres.Use do
         def create_type() do
           Ecto.Migration.execute(unquote(create_sql), unquote(drop_sql))
         end
+
+        def alter_type() do
```
Comment (gjaldon) [inline lib/ecto_enum/postgres/use.ex]: Hi @sofakingworld. Can we rename the function to `add_value`? `alter_type` can be a bunch of different things like renaming the type.

**Review by gjaldon — CHANGES_REQUESTED** [review]

**Comment by gjaldon** [comment]
Thanks a lot @sofakingworld 💜 🎉 💜

**Pull request merged**